src/frontend/Komponenter/Behandling/Totrinnskontroll/utils.test.ts

```diff
@@ -52,9 +52,9 @@ describe('Har barn mellom seks og tolv måneder', () => {
         expect(harBarnMellomSeksOgTolvMåneder(vilkår as IVilkår)).toBe(false);
     });
 
-    it('skal returnere false når et barn er over 12 måneder', () => {
+    it('skal returnere false når et barn er 12 måneder eller mer', () => {
```

Comment:
```suggestion
    it('skal returnere false når et barn er 12 måneder eller eldre', () => {
```
Hørtes litt rart ut med "mer" :D

Reply:
Enig, eldre blir mer riktig!